scripts/uninstall.sh

```diff
@@ -71,6 +71,9 @@ else
   exit 1
 fi
 
+# Lire la version depuis le fichier version.txt
```

Comment:
Please properly translate this to English.

Reply:
Good catch